### PR TITLE
Ai hints cleanup

### PR DESCRIPTION
**Pull request by rubenk**

### CODE REVIEW

File: src/chrony.c [hunk 1]
```diff
@@ -306,14 +306,14 @@ static int
 connect_client(const char *p_hostname,
                const char *p_service, int p_family, int p_socktype)
 {
-  struct addrinfo hints = { 0 };
-  struct addrinfo *res = NULL, *ressave = NULL;
+  struct addrinfo ai_hints = { 0 };
```
Comment (octo) [inline src/chrony.c]: Why is this struct initialized differently? Looks like the other schema would work here, too.

File: src/write_sensu.c [hunk 2]
```diff
@@ -115,9 +115,7 @@ static int sensu_connect(struct sensu_host *host) /* {{{ */
 		host->res = NULL;
 		hints.ai_family = AF_INET;
 		hints.ai_socktype = SOCK_STREAM;
-#ifdef AI_ADDRCONFIG
-		hints.ai_flags |= AI_ADDRCONFIG;
-#endif
+		hints.ai_flags = AI_ADDRCONFIG;
```
Comment (octo) [inline src/write_sensu.c]: You rename the variable to `ai_hints` above. Could you unify this, i.e. change this here, too?

Comment (octo) [inline src/write_sensu.c]: Looking at the context, it looks like `hints` is only used within the `if (!READY)` block. Can we move it into the if block?

**Comment by octo** [comment]
LGTM, but please note inline comments.

File: src/chrony.c (hunk 1 above)
Comment (rubenk) [inline src/chrony.c]: Somehow I missed that, will fix.

File: src/write_sensu.c (hunk 2 above)
Comment (rubenk) [inline src/write_sensu.c]: Nice catch!

**Pull request merged**